packages/ember-test-app/app/templates/marketing.hbs

```diff
@@ -203,4 +203,31 @@
       </div>
     </form>
   </div>
-</main>
\ No newline at end of file
+</main>
+
+<Mktg::Footer @hasHorizontalLine={{true}}>
+  <:nav>
+    <Button class="ms-2 rounded-pill btn-light">Get Started</Button>
+    <a href="" class="text-light">Pricing &amp; Plans</a>
```

Comment:
Should we add these classes just so we have an example of not underlining the links? `link-underline link-underline-opacity-0`